src/ywtListView.js

```diff
@@ -66,18 +48,89 @@ kintone.events.on('app.record.index.show', (event) => {
       // プロフィール画像取得
       for (const o of optRecords) {
         if (r.mail.value === o.mail.value) {
-          const fileKey = o.profile_img.value.length > 0 ? o.profile_img.value[0].fileKey : undefined;
+          const fileKey = o.profile_img.value.length > 0 ? o.profile_img.value[0].fileKey : false;
           if (fileKey) {
             fileDownload(fileKey, i);
           }
         }
-      };
+      }
       return r;
     });
 
     new Vue({
       // Vueを適用するelement
       el: '#customize_view',
+      methods: {
+        showDetail: function (record, rowIndex) {
+          // 行選択表示
+          const rows = document.querySelectorAll('.list_row');
+          rows.forEach(function (row, i) {
+            if (i == rowIndex) {
+              row.classList.add('row_selected');
+            }
```

Comment:
ERROR: Closing curly brace does not appear on the same line as the subsequent block. (brace-style)